jenkins/public_tests_all.Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
     stage('Local Build') {
       agent any;
       steps {
-        sh "./build_openroad.sh --local";
+        sh "./build_openroad.sh --local --openroad-args -DCMAKE_CXX_FLAGS=-fsanitize=undefined";
```

Comment:
@vvbandeira FYI, there are three types of sanitzation: undefined, address and memory. The latter is only supported by LLVM.